## Record of work, 20140519

 1. Added logging to `requests` (tested in `make_urlrequest()`, one of three `try` blocks in this module).
 1. Added logging to `city_codes` (not yet tested, but also calls `requests.make_urlrequest()`, already tested).
 1. Must still do all other modules containing `try-except` blocks:
   2. `map_us.py`
   2. `populate_db.py`
   2. `retrieve.py`
   2. `utils.py`
 1. `requests` now has `main()` which can be called from command line; this will aid setting up `cron` jobs soon.

[end]
